tree-construction/template.dat

```diff
@@ -1625,3 +1625,39 @@ template
 |         name="q"
 |     <div>
 |       "second"
+
+#data
+<!DOCTYPE HTML><template><tr><td>cell</td></tr></template>
+#document-fragment
+template
+#errors
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|     <template>
+|       content
+|         <tr>
+|           <td>
+|             "cell"
+|   <body>
+
+#data
+<!DOCTYPE HTML><template> <tr> <td>cell</td> </tr> </template>
+#document-fragment
+template
+#errors
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|     <template>
+|       content
+|         " "
+|         <tr>
+|           " "
+|           <td>
+|             "cell"
+|           " "
+|         " "
+|   <body>
```

Comment:
There should be a LF before the end of the file: https://github.com/html5lib/html5lib-tests/blob/master/tree-construction/README.md